package.json

```diff
@@ -358,5 +358,8 @@
     },
     "msw": {
         "workerDirectory": [".storybook/public"]
+    },
+    "dependencies": {
+        "box-typescript-sdk-gen": "^1.17.1"
     }
```

Comment:
nit: i would move this the `dependencies` section to be near peer and dev. either put it before `devDependencies` or after `peerDependencies`